README.md

```diff
@@ -209,6 +209,30 @@ The information for external packages is stored in
           zlib@1.2.11 +optimize+pic+shared: /gpfs/bbp.cscs.ch/apps/hpc/jenkins/deploy/external-libraries/2019-01-04/linux-rhel7-x86_64/gcc-6.4.0/zlib-1.2.11-w43e56tzqj
 </details>
 
+## Building software in MAC
+
+<details>
+<summary>
+Install software in MAC. Use BREW as a source of binary packages
+</summary>
+
+In MAC the build process is very similar to Ubuntu. To avoid building
+the whole stack from source we can likewise use a package manager.
+For that end we have successfully used Brew. We also provide a skeleton 
+packages.yaml that you should review and adapt to your needs.
+
+Before starting, please install brew and the required packages.
+If you require Python please install a version dowloaded from
+Python.org, as several issues have been found with "brewed" Python
```

Comment:
Thanks for getting this started! I'm thinking we could probably move this section up underneath the Ubuntu ones?

```suggestion
## Building software on OS X

<details>
<summary>
Install software on OS X, using Homebrew for binary packages
</summary>

On OS X the build process is very similar to Ubuntu. To avoid building
the whole stack from source we can likewise use another package manager to provide precompiled binaries.
To that end we have successfully used Homebrew. We also provide a skeleton 
`packages.yaml` that you should review and adapt to your needs.

Before starting, please install brew and the required packages.
If you require Python please install a version dowloaded from
Python.org, as several issues have been found with Homebrew's Python
```
Which packages are required? Also, rather than downloading an external Python, Spack could just build it…

Reply:
> Which packages are required? Also, rather than downloading an external Python, Spack could just build it…

Not a bad idea, if we nail all its dependencies beforehand, takes ~6 mins to build. @ferdonline what do you say about adopting this strategy? Brew's Python is also causing issues with wheels as far as you were pointing out.

Reply:
The issue is not the build time, even though is not negligible. The issue is that it will probably end up with the same issues as brew Python... Python from Python.org are the standard ones, build with well tested settings and env vars for retro versions compatibility. I think it should be the default. Mac users only have to download and install, not like the linux world.